commands/base_commands/social.py

```diff
@@ -984,7 +984,7 @@ def func(self):
             self.set_or_remove_retainer_ability(attr_name, attr_desc)
             return
         # get the first new messenger we have waiting
-        if "receive" in self.switches:
+        if "receive" or "recieve"in self.switches:
```

Comment:
```suggestion
        if self.check_switches(["receive", "recieve"]:
```
Need to have either be `in self.switches` or it's just checking for whether a string literal is Truthy. Because of how verbose/annoying that gets, I made the `check_switches` method as just a quick set comparison between `self.switches` and any iterable you pass to it.

You're right that messenger is a mess so I don't blame you for not wanting to mess with it, though.